### PR TITLE
Add modifications to support PC style home and end

### PR DESCRIPTION
**Pull request by ktaragorn**
add 2 different pairs of modifications since mac offers multiple ways of doing nearly the same thing

### CODE REVIEW

**Comment by ktaragorn** [comment]
Fixes #90

File: docs/json/pc_shortcuts.json [hunk 1]
```diff
@@ -2,6 +2,102 @@
...
+              "modifiers": [
+                "left_control"
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "description": "End key to beginning of line (Control + e)",
```
Comment (muyinliu) [inline docs/json/pc_shortcuts.json]: `"description": "End key to end of line (Control + e)",`

File: docs/json/pc_shortcuts.json [hunk 2]
```diff
@@ -2,6 +2,102 @@
...
+              "key_code": "e",
+              "modifiers": [
+                "left_control"
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
```
Comment (muyinliu) [inline docs/json/pc_shortcuts.json]: conflict with `"Home key to beginning of line (Control + a)"` ?

File: docs/json/pc_shortcuts.json (hunk 1 above)
Comment (ktaragorn) [inline docs/json/pc_shortcuts.json]: Forgot to make after fixing the issue

File: docs/json/pc_shortcuts.json (hunk 2 above)
Comment (ktaragorn) [inline docs/json/pc_shortcuts.json]: What do you mean?

Comment (muyinliu) [inline docs/json/pc_shortcuts.json]: home -> control + A
home -> command + left_arrow

conflict.

Comment (ktaragorn) [inline docs/json/pc_shortcuts.json]: 2 options, hence the different wording. Seems like mac has 2 different definitions for these. Control + A - begining of the line including whitespace, command  + <- is begining of the sentence, excluding whitespace.

Comment (ktaragorn) [inline docs/json/pc_shortcuts.json]: Not a conflict cuz ppl can decide which they want.

File: docs/json/pc_shortcuts.json (hunk 1 above)
Comment (ktaragorn) [inline docs/json/pc_shortcuts.json]: Fixed!

File: docs/json/pc_shortcuts.json (hunk 2 above)
Comment (muyinliu) [inline docs/json/pc_shortcuts.json]: OK.

**Pull request merged**